public/index.html

```diff
@@ -5,6 +5,10 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
```

Comment:
❔ Should we remove this? Leave it in? change it to
```suggestion
    <meta
      name="description"
      content="Web site created using spraygun"
    />
```

Reply:
I think we should take it out. It would be weird for that to go into production accidentally, and then you find out when someone shares your app on social and that description pop up 😄

Reply:
👍